src/main/groovy/me/champeau/gradle/JapicmpAbstractTask.groovy

```diff
@@ -93,9 +98,11 @@ abstract class JapicmpAbstractTask extends AbstractTask {
         def options = new Options()
         options.outputOnlyModifications = onlyModified
         options.setAccessModifier(AccessModifier.valueOf(accessModifier.toUpperCase()))
+        options.htmlOutputFile = GOptional.fromNullable(htmlOutputFile?.absolutePath)
```

Comment:
If you think it can bring problems I can make a PR to japicmp to provide additional setter which takes `String`.